aries-site/src/data/wcag/datasearch.json

```diff
@@ -0,0 +1,512 @@
+  },
+  {
+    "rule": "2.4.11",
+    "label": "Focus Not Obscured (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.12",
+    "label": "Focus Not Obscured (Enhanced)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.4.13",
+    "label": "Focus Appearance",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.1",
+    "label": "Pointer Gestures",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.2",
+    "label": "Pointer Cancellation",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.3",
+    "label": "Label in Name",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.4",
+    "label": "Motion Actuation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.5",
+    "label": "Target Size (Enhanced)",
+    "status": "failed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.6",
+    "label": "Concurrent Input Mechanisms",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.7",
+    "label": "Dragging Movements",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "2.5.8",
+    "label": "Target Size (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.1",
+    "label": "Language of Page",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.2",
+    "label": "Language of Parts",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.3",
+    "label": "Unusual Words",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.4",
+    "label": "Abbreviations",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.5",
+    "label": "Reading Level",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.6",
+    "label": "Pronunciation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.1",
+    "label": "On Focus",
+    "status": "conditional",
+    "notes": "when using the defaultSuggestion prop this will fail so this prop should be used with caution."
+  },
+  {
+    "rule": "3.2.2",
+    "label": "On Input",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.3",
+    "label": "Consistent Navigation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.4",
+    "label": "Consistent Identification",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.5",
+    "label": "Change on Request",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.6",
+    "label": "Consistent Help",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.1",
+    "label": "Error Identification",
+    "status": "not-applicable",
+    "notes": "Dependent on FormField."
+  },
+  {
+    "rule": "3.3.2",
+    "label": "Labels or Instructions",
+    "status": "not-applicable",
```

Comment:
I think this rule is applicable and we are passing because we have a visual indication that this is a search field with the icon we are using.

Relevant wcag info: 

> A website provides a global search field in the header of the site. Any term can be entered, so there are no instructions needed, but the field needs a cue to communicate its purpose. Commonly, such search field will be paired with a "loupe" or "magnify glass" search icon, serving as its visible label, if not also doubling as the visual identifier for the button that submits the search query.